kovri.md

```diff
@@ -32,13 +32,14 @@ So what can Alice do to mitigate these threats? Well, she can use Kovri!
 If Alice sends her transaction with Kovri, nodes will no longer see her IP address, making passive surveillance tactics impractical.
 
 Now let’s imagine a different scenario.
-Suppose Charlie has an Internet Service Provider that does not want him to use Monero and he receives a warning to cease and desist in the mail.
-Charlie could optionally mask his Internet traffic through Kovri and in doing so, prevent others from knowing he is running a Monero node.
+Suppose Alex wants to support the Monero network by running a full node at her home.
```

Comment:
Please don't change the name. See https://github.com/alvinjoelsantos/promo-video/pull/25#discussion_r135118023.